roles/network_plugin/calico/templates/calico.env.j2

```diff
@@ -4,6 +4,11 @@ ETCD_CERT_FILE="{{ calico_cert_dir }}/cert.crt"
 ETCD_KEY_FILE="{{ calico_cert_dir }}/key.pem"
 CALICO_IP="{{ip | default(ansible_default_ipv4.address) }}"
 CALICO_IP6=""
+{% if calico_network_backend is defined and calico_network_backend == 'none' %}
```

Comment:
More of a question. Should a default var `calico_network_backend` be defined and set to `none` as a recommended approach with this change? Looking at [docs](https://docs.projectcalico.org/v2.4/reference/node/configuration) I don't see what this string is used for.